pkg/ccl/changefeedccl/event_processing.go

```diff
@@ -538,7 +591,16 @@ func (c *parallelEventConsumer) workerLoop(
 			defer c.mu.Unlock()
 			return c.mu.termErr
 		case e := <-c.workerCh[id]:
+			// Request CPU time to use for event consumption, block if this time is
+			// unavailable. If there is unused CPU time left from the last call to
+			// Pace, then use that time instead of blocking.
+			if err := pacer.Pace(ctx); err != nil {
```

Comment:
@irfansharif should review this usage -- if it's sane, etc (it does look okay to me).